src/json.d

```diff
@@ -619,7 +619,22 @@ public:
         jsonProperties(d);
         TypeFunction tf = cast(TypeFunction)d.type;
         if (tf && tf.ty == Tfunction)
+        {
             property("parameters", tf.parameters);
+            property("trust", tf.trust);
+            propertyBool("nothrow", tf.isnothrow);
+            propertyBool("nogc", tf.isnogc);
```

Comment:
It might be worthwhile to put these under `attributes`, especially considering that UDAs should also be part of the output.